CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+cmake_minimum_required(VERSION 3.0)
+project(durian)
+
+set(CMAKE_CXX_STANDARD 14)
+
+include_directories(${PROJECT_SOURCE_DIR}/src/include)
```

Comment:
Shouldn't we `.gitignore` the cmake output?

Reply:
I'd include the executables after `make` is ran as well.

Reply:
Yeah, we should. I was also going to include IDE files but wanted by to ask around what IDEs y’all were planning to use first.

Reply:
VScode is life sir

Reply:
VSCode

Reply:
Do you guys happen to know what IDE files VScode uses?